cognite_toolkit/_cdf_tk/client/api/extended_timeseries.py

```diff
@@ -90,6 +96,68 @@ def _set_pending_ids(self, identifiers: Sequence[PendingInstanceId]) -> Extended
 
         return ExtendedTimeSeriesList._load(retrieved_items, cognite_client=self._cognite_client)
 
+    @overload
+    def unlink_instance_ids(
+        self,
+        id: int | None = None,
+        external_id: str | None = None,
+    ) -> ExtendedTimeSeries | None: ...
+
+    @overload
+    def unlink_instance_ids(
+        self,
+        id: Sequence[int] | None = None,
+        external_id: SequenceNotStr[str] | None = None,
+    ) -> ExtendedTimeSeriesList: ...
+
+    def unlink_instance_ids(
+        self,
+        id: int | Sequence[int] | None = None,
+        external_id: str | SequenceNotStr[str] | None = None,
+    ) -> ExtendedTimeSeries | ExtendedTimeSeriesList | None:
+        """Unlink pending instance IDs from time series.
+
+        Args:
+            id (int | Sequence[int] | None): The ID(s) of the time series.
+            external_id (str | SequenceNotStr[str] | None): The external ID(s) of the time series.
+
+        """
+        if id is None and external_id is None:
+            return None
+        if isinstance(id, int) and isinstance(external_id, str):
+            raise ValueError("Either id or external_id must be provided, or both as a sequence.")
```

Comment:
This message is a bit unclear, at least I don't read `either` as strictly not both.